AFQ/tests/test_roi.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
```

Comment:
Maybe we should have a tests folder in the new module (which will be called "recognition"?)